pkg/api/types.go

```diff
@@ -181,10 +198,13 @@ type Binding struct {
 	Status BindingStatus
 }
 
+// BindingSpec represents a description of a Binding.
 type BindingSpec struct {
 	// InstanceRef is the reference to the Instance this binding is to.
+	// Immutable.
 	InstanceRef kapi.ObjectReference
-	// AppLabelSelector selects the pods in the Binding's namespace that should be injected with the results of the binding
+	// AppLabelSelector selects the pods in the Binding's namespace that
+	// should be injected with the results of the binding.  Immutable.
 	AppLabelSelector kapi.LabelSelector
```

Comment:
The reference was removed because the object doesn't yet exist.  We need to design the API at first to not have it, since it's likely that we will implement the API before the object exists in the core.